packages/service/package.json

```diff
@@ -26,10 +29,20 @@
     "watch": "pnpm run build:tsc -w"
   },
   "dependencies": {
+    "@wdio/globals": "^8.36.1",
     "@wdio/logger": "^8.28.0",
     "@wdio/types": "^8.36.1",
-    "@wdio/globals": "^8.36.1",
+    "fuse.js": "^7.0.0",
+    "inquirer": "^9.2.20",
+    "jimp": "^0.22.12",
+    "node-tesseract-ocr": "^2.2.1",
+    "tesseract.js": "^5.0.5",
```

Comment:
I wonder if this doesn't blow the visual service unnecessarily larger than it has to be. How many people will actually use only the visual part and don't need all these additional OCR related dependencies? Why not have a `@wdio/ocr-service`?

Reply:
The two downsides to keep this in a single package for me are:

- extra dependencies added (some are over 1.4mb) making a direct impact to setup times
- extra commands that can get users confused that just want to do visual comparison testing

I think by creating an extra package for OCR still keeps everything within a single place (this repository). The question really is: how many people tend to use visual + OCR features __vs__ how many people only want to do visual regression testing or are only interested in OCR features.

How about the following compromise:
- we keep `@wdio/visual-service` as is without OCR primitives
- we create a `@wdio/ocr-service` with all OCR primitives
- we create a `@wdio/visual-ocr-service` that is a very slim package that has both packages as dependency and exports a service that executes both services

Reply:
hmm, I'm not that worried about the size, people add more npm modules on the fly (showing "hello world" with a default Angular setup already downloads 450+ mb 😅 ). 
Regarding the commands, I'm also not that worried about it, it's just about how we document it.

I can follow you from a developer point of few, but I believe that having this in one Visual service results in less complexity for most of our endusers

Reply:
> default Angular setup already downloads 450+ mb 😅 

Yes, however if you just install WebdriverIO with visual 1.4MB will be enough to "feel" a difference. Also these dependencies come with their own dependencies so the total added amount of payload is probably much higher.

From another point of view: the point of mono repos is its ability to nicely separate large code bases into separate sub projects. I feel this gives every type of feature its own little space and allows folks to contribute to certain areas without having to understand the whole picture. Keeping OCR in the visual package requires folks to understand both code bases to some extend while keeping them separate makes things easier a bit, especially if someone wants to fix a bug in OCR.

Note that we don't know yet how much we will add to the OCR feature in the future.

We could have build the whole WDIO testrunner within a single NPM package, all services, reporters and utilities in one. This would probably caused a lot of interdependencies that would have made the code base more complex than it is today.

>  I believe that having this in one Visual service results in less complexity for most of our endusers

This assumes that every visual user will use OCR features which is where I disagree. I think there will be plenty of people that want to do just visual comparison testing of their components or apps in general. On the other side, I am sure there will be people only interested in the OCR feature to e.g. to some automation on a canvas. And for folks who actually want to use both, they can just install `@wdio/visual-ocr-service`